### DISCUSSION TITLE
No está el contenido en el github de Backend??

### DISCUSSION THREAD

**Discussion by LicesDev** [Launch-X-Latam/MisionFrontEnd#473] [Team Posts]
No puedo observar el contenido de Backend

**Comment by loo-kuhs** [comment]
Te refieres a -> [este contenido](https://github.com/loo-kuhs/MisionBackend-LaunchX/tree/d4286e7798e19384fba030f93adf93593e3969d7)?

**Comment by AriadnaDalVqz** [comment]
X2 yo tampoco puedo verlo

**Comment by Vitaliquid** [comment]
se hizo el aviso de que el contenido y el temario sufrieron cambios para poner al corriente a los explorers. tengan paciencia. vean los lives y más adelante estará disponible el contenido.

**Comment by mariocanul30** [comment]
Ya esta el contenido, ayer se termino de subir, aunque dijo el sherpa que ira agregando mas conforme avancemos

**Comment by Leonardo-Ismet** [comment]
El repositorio si cuenta con la información, aunque no se está actualizando de forma automática y deben hacerlo de forma manual, para esto si no tienen el fork solo lo crean, ahora si ya tienen el fork, buscan la barra de notificaciones donde avisa sobre los commits, al final de esa barra aparece una pestaña denominada sync fork, presionan y luego presionan en el botón Update branch y se actualizara la información, hay que realizar esto con cada actualización, dejo una imagen de referencia, espero le ayude, saludos.

![204857720-bb559ea8-9a0b-435a-89ed-94db5bccf8c9](https://user-images.githubusercontent.com/88945000/204864984-bbaf979b-9e62-4c08-abdd-d64d6b3ae776.png)
